subliminal/cli.py

```diff
@@ -97,7 +97,11 @@ def convert(self, value, param, ctx):
 def subliminal(ctx, addic7ed, cache_dir, debug):
     """Subtitles, faster than your thoughts."""
     # create cache directory
-    os.makedirs(cache_dir, exist_ok=True)
+    try:
+        os.makedirs(cache_dir)
+    except OSError:
+        if not os.path.isdir(cache_dir):
```

Comment:
I believe this part is not necessary as it's already done by click. You can just pass.